server/api/scheduler.go

```diff
@@ -324,6 +324,36 @@ func (h *schedulerHandler) PauseOrResume(w http.ResponseWriter, r *http.Request)
 	h.r.JSON(w, http.StatusOK, "Pause or resume the scheduler successfully.")
 }
 
+// FIXME: details of input json body params
+// @Tags checker
+// @Summary Pause or resume region merge.
+// @Accept json
+// @Param name path string true "The name of the scheduler."
+// @Param body body object true "json params"
+// @Produce json
+// @Success 200 {string} string "Pause or resume the scheduler successfully."
+// @Failure 400 {string} string "Bad format request."
+// @Failure 500 {string} string "PD server failed to proceed the request."
+// @Router /checker/{name} [post]
+func (h *schedulerHandler) PauseOrResumeChecker(w http.ResponseWriter, r *http.Request) {
```

Comment:
I think using `checkerHandler` will be better

Reply:
Function renamed.